src/get-authorization-url.ts

```diff
@@ -1,12 +1,16 @@
 import { workos } from './workos.js';
 import { WORKOS_CLIENT_ID, WORKOS_REDIRECT_URI } from './env-variables.js';
+import { GetAuthURLOptions } from './interfaces.js';
+
+async function getAuthorizationUrl(options: GetAuthURLOptions) {
```

Comment:
Shall we make this optional?

```suggestion
async function getAuthorizationUrl(options: GetAuthURLOptions = {}) {
```